src/bp/ml/svm-js/util/split-dataset.js

```diff
@@ -34,16 +34,18 @@ module.exports = function(dataset, kFold) {
     return subset
   })
 
-  return _(nIndexes).map(function(a, i, list) {
-    return {
-      test: sets[i],
-      train: _.chain(list)
-        .without(i)
-        .map(function(iii) {
-          return sets[iii]
-        })
-        .flatten()
-        .value()
-    }
-  })
+  return _(nIndexes)
+    .map(function(a, i, list) {
+      return {
+        test: sets[i],
+        train: _.chain(list)
+          .without(i)
+          .map(function(iii) {
+            return sets[iii]
+          })
+          .flatten()
+          .value()
+      }
+    })
+    .value()
 }
```

Comment:
use implicit returns when there's no state?
_(nIndexes)
    .map((a, i, list) => ({
        test: sets[i],
        train: _.chain(list)
          .without(i)
          .map((idx) => sets[iii])
          .flatten()
          .value()
    })
    .value()

Also, since we're not using any lodash function on the top level loop we don't need to wrap nIndexes;
nIndexes.map((a, i, list) => ({
    test: sets[i],
    train: _.chain(list)
      .without(i)
      .map((idx) => sets[idx])
      .flatten()
      .value()
}))